RNA-seq/setup/04-nb_cell_line_tximeta.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "NB Cell Line: gene-level summarization with `tximeta`"
```

Comment:
This is 99.99% the same as `02-gastric_cancer_tximeta.Rmd` which was reviewed in #343. I search and replaced `gastric-cancer` with `NB-cell` and then had to add `..` so it would run in `setup`.